contrib/babelfishpg_tsql/src/pl_exec.c

```diff
@@ -4612,6 +4612,9 @@ exec_stmt_execsql(PLtsql_execstate *estate,
 
 	/* fetch current search_path */
 	char	   *old_search_path = NULL;
+	bool		ro_func = (estate->func->fn_prokind == PROKIND_FUNCTION) &&
```

Comment:
let us simply the condition at other place to use this one as well

Reply:
Done